src/megachatapi_impl.cpp

```diff
@@ -1175,6 +1175,44 @@ int MegaChatApiImpl::getInitState()
     return initState;
 }
 
+bool MegaChatApiImpl::enableChat(const char *sid)
+{
+    if (sid == NULL)
```

Comment:
Consider to check `sid[0] != '/0'`. Maybe it's a good protection, I've seen an `assert(sid.empty())` in `openDb()`